tests/Fixer/Operator/NoUselessConcatOperatorFixerTest.php

```diff
@@ -0,0 +1,176 @@
+        ];
+
+        foreach ($cases as $label => $case) {
+            yield $label => [
+                sprintf($templateExpected, $case[2]),
+                sprintf($templateInput, $case[0], $case[1]),
+            ];
+        }
+
+        yield 'encapsed followed by simple double quoted 1' => [
+            '<?php echo "Hello, {$fruit}s.";',
+            '<?php echo \'Hello,\'  .  " {$fruit}s.";',
+        ];
+
+        yield 'encapsed followed by simple double quoted 2' => [
+            '<?php echo "Hello.He drank some juice made of {$fruit}s.Bye $user!" /*1*//*2*/ /*3*//*4*/;',
+            '<?php echo \'Hello.\' /*1*/ . /*2*/ "He drank some juice made of {$fruit}s."/*3*/./*4*/"Bye $user!";',
+        ];
+
+        yield [
+            '<?php
+$string = "foobar";
+echo "char @ -4 \"[$string[-2]]\"!";
+',
+            '<?php
+$string = "foobar";
+echo "char @ -4 \"[$string[-2]"."]\"!";
+',
+        ];
+
+        yield 'double quote concat double quote + comment' => [
+            '<?php $fi = "lk" /* 1 *//* 2 */ ;',
+            '<?php $fi = "l" /* 1 */ . /* 2 */ "k";',
+        ];
+
+        yield 'empty concat empty' => [
+            '<?php $aT = "";',
+            '<?php $aT = ""."";',
+        ];
+
+        yield 'multiple fixes' => [
+            '<?php $f0 = "abc | defg";',
+            '<?php $f0 = "a"."b"."c | "."d"."e"."f"."g";',
+        ];
+
+        yield 'linebreak with indent inside' => [
+            '<?php
+$text1 = "intro:   |   |"."
+               line 2 indent";',
+            '<?php
+$text1 = "intro:   |"."   |"."
+               line 2 indent"."";',
+        ];
+
+        yield 'linebreak with indent inside + comment' => [
+            '<?php
+$text2 = "intro:      "." #a
+               line 2 indent";',
+            '<?php
+$text2 = "intro:   "."   "." #a
+               line 2 indent"."";',
+        ];
+
+        yield 'do not fix' => [
+            '<?php
+                $a0x = $b . "c";
+                $a1x = "c" . $b;
+                $b2x = foo() . "X";
+                $b3x = foo() . \'Y\';
+                $b4x = "Z" . foo();
+                $b5x = \'b\' . foo();
+                $b6x = \'X  \n \' . "\n\t";
+                $b7x = "\n\t" . \'X $a\';
+                $b7x = "abc". 1;
+                $b7x = "def". 1.2;
+                // bin string
+                $f202 = b"a"."b";
+                $f201 = b"a".b"b";
+                $f203 = "a".B"b";
+                echo b"He drank some juice made of {$fruit}s.".b" Sliced the {$fruit}s.";
+            ',
+        ];
+
+        yield 'single quote concat single quote but with line break after' => [
+            "<?php \$fh = 'x'. // some comment
+'y';",
+        ];
+
+        yield 'single quote concat single quote but with line break before' => [
+            "<?php \$ff = 'x' // some comment
+.'y';",
+        ];
+
+        yield 'linebreak without indent inside' => [
```

Comment:
Why not fixing this one?

Reply:
The fixer doesn't touch any line break cases. After testing the fixer on the 50 PHP projects I found that most projects use line breaks in combination with concat operations for readability by splitting long strings this way. 
As such I don't think these should be touched.